db/migrate/20170223193040_add_workflow_ref_to_permission_templates.rb

```diff
@@ -0,0 +1,5 @@
+class AddWorkflowRefToPermissionTemplates < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :permission_templates, :workflow, foreign_key: true
```

Comment:
Is this https://github.com/projecthydra-labs/hyrax/pull/488 (should be merged any minute) going to change what's being done here? Just curious.